plugins/contact-resources/src/components/PersonContent.svelte

```diff
@@ -15,35 +15,125 @@
 <script lang="ts">
   import { formatName, Person } from '@anticrm/contact'
   import { Hierarchy } from '@anticrm/core'
+  import { IntlString } from '@anticrm/platform'
   import { Avatar } from '@anticrm/presentation'
-  import { getPanelURI } from '@anticrm/ui'
+  import { getPanelURI, Label } from '@anticrm/ui'
   import view from '@anticrm/view'
 
   export let value: Person | undefined
   export let inline: boolean = false
+  export let isInteractive = true
   export let shouldShowName = true
   export let shouldShowPlaceholder = false
-
-  const avatarSize = 'x-small'
+  export let defaultName: IntlString | undefined = undefined
+  export let avatarSize: 'inline' | 'tiny' | 'x-small' | 'small' | 'medium' | 'large' | 'x-large' = 'x-small'
+  export let onEdit: ((event: MouseEvent) => void) | undefined = undefined
 </script>
 
-{#if value || shouldShowPlaceholder}
+{#if !isInteractive}
   {#if value}
+    <div class="contentPresenter mContentPresenterNotInteractive">
+      <div class="eContentPresenterIcon">
+        <Avatar size={avatarSize} avatar={value.avatar} />
+      </div>
+      {#if shouldShowName}
+        <span class="eContentPresenterLabel">{formatName(value.name)}</span>
+      {/if}
+    </div>
+  {:else}
+    <div class="contentPresenter mContentPresenterNotInteractive">
+      <div class="eContentPresenterIcon">
+        <Avatar size={avatarSize} avatar={undefined} />
+      </div>
+      {#if defaultName}
+        <div class="eContentPresenterLabel">
+          <Label label={defaultName} />
+        </div>
+      {/if}
+    </div>
+  {/if}
+{:else if value || shouldShowPlaceholder}
+  {#if onEdit}
+    <div class="contentPresenter" class:inline-presenter={inline} on:click={onEdit}>
+      <div class="eContentPresenterIcon">
+        <Avatar size={avatarSize} avatar={value?.avatar} />
+      </div>
+      {#if value && shouldShowName}
+        <span class="eContentPresenterLabel">{formatName(value.name)}</span>
+      {/if}
+    </div>
+  {:else if value}
     <a
```

Comment:
Please use https://svelte.dev/tutorial/svelte-element and you could simplify logic without having duplicates.

Reply:
Done.